### PR TITLE
Small docs/imgs fixes, and fixed broken links

### PR DESCRIPTION
**Pull request by planteater**
- Update anomaly detection docs, including image
- Update performance monitoring image
- Update buildSHA image
- Move Code Level Metrics article to bottom of its section in left nav
- Update broken links to our repo association and build SHA info

### CODE REVIEW

File: src/content/docs/apm/agents/go-agent/configuration/go-agent-configuration.mdx [hunk 1]
```diff
@@ -612,7 +612,7 @@ In this and the following examples, `config` represents your New Relic config st
   </Collapser>
 </CollapserGroup>
 
-If you're using New Relic CodeStream to monitor performance from your IDE you may also want to [associate repositories with your services](/docs/codestream/how-use-codestream/performance-monitoring/#repo-association) and [associate build SHAs or release tags with errors](/docs/codestream/how-use-codestream/performance-monitoring/#buildsha).
+If you're using New Relic CodeStream to monitor performance from your IDE you may also want to [associate repositories with your services](/docs/codestream/how-use-codestream/observability/repo-association) and [associate build SHAs or release tags with errors](/docs/codestream/observability/error-investigation/#buildsha).
```
[the remark this answers is not in the document]
Comment (rhetoric101) [reply src/content/docs/apm/agents/go-agent/configuration/go-agent-configuration.mdx:615]: @planteater I noticed that for other agents, you're inserting this comment under environment variables. I just wanted to confirm you wanted to put it under "General configurations" here in Go. Well, I just noticed that for PHP, it's under "General configurations" so you can probably disregard my comment.

File: src/content/docs/codestream/observability/transactions.mdx [hunk 2]
```diff
@@ -15,10 +15,12 @@ CodeStream automatically flags performance problems in your code, allowing you t
 
 CodeStream identifies performance issues by starting with transactions and looking for anomalies in either error rate or average duration. If you're using New Relic's [change tracking](/docs/change-tracking/change-tracking-introduction/) to send deployment information, anomalies are calculated by comparing the performance since your last release against the preceding three weeks. Otherwise, the comparison is made using the last seven days. For any transactions identified as having performance issues, CodeStream then drills down into the segments (e.g., database queries, external calls, individual methods, etc.) that make up the transaction to see if any of them are performing poorly. This allows you to hone in on the source of the problem.
 
-Click on any transaction or segment to see the performance issue charted over time, and CodeStream, when possible, we also take you to the appropriate file and line number.
+Click on any transaction or segment to see the performance issue charted over time. For anomalous error rates, the errors contributing to the increased error rate are listed and you can click on one to go into CodeStream's [error investigation]() experience.
```
Comment (rhetoric101) [inline src/content/docs/codestream/observability/transactions.mdx:18]:
```suggestion
Click on any transaction or segment to see the performance issue charted over time. For anomalous error rates, the errors contributing to the increased error rate are listed and you can click on one to go into CodeStream's [error investigation](/docs/codestream/observability/error-investigation) experience.
```

**Review by rhetoric101 — COMMENTED** [review]
@planteater thanks for updating these links. I inserted just a few comments. Also, I don't see updates to Java or Ruby configuration docs. Are these not supported?

**Comment by planteater** [comment]
> @planteater thanks for updating these links. I inserted just a few comments. Also, I don't see updates to Java or Ruby configuration docs. Are these not supported?

Hi @rhetoric101 ! The various agent teams handled their own docs, and incorporated CodeStream/code-level metrics in different ways. So not all agents had the links that needed to be updated.

**Review by rhetoric101 — APPROVED** [review]
@planteater this looks good to me!

**Pull request merged**